src/app/maker/maker-pending-shift-condensed/maker-pending-shift-condensed.component.html

```diff
@@ -2,10 +2,10 @@
   <ion-card width="100%" padding="5px" color="light">
     <mat-expansion-panel>
       <mat-expansion-panel-header>
-        <mat-panel-title color="warning">6/6 FILLED
+        <mat-panel-title color="warning">
           <ion-icon name="alert" color="warning"></ion-icon>
-          <ion-label><b>{{shift.name}} </b>
-            <div class-name="shift-time_date"> {{shift.time_date | date:'MMM dd, yyyy h:mm a'}}</div>
+          <ion-label><b>{{shift.name}}</b> @ {{shift.start | date:'MMM dd, yyyy h:mm a'}}
+          <div class-name="shift-time_date"> Positions Filled: {{count}} out of {{shift.positions.length}}</div>
```

Comment:
Pending shifts should already have all positions filled, right? So I think we could forgo displaying that count at all on those.